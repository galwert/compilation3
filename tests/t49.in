// this is oke :)
int func() {
    return 1;
}

int ags() {
    return 1;
}

int zxvc() {
    return 1;
}

int sdg() {
    return 1;
}

int qwr(bool x1, bool x2) {
    return 1;
}
void main(){
    printi(sdg());
    printi(qwr(true, false));
    printi(zxvc());
}


bool sdfa4(bool x1, bool x2) {
    return x1 or x2;
}

void nbfdrct(bool x1, bool x2) { 
    int x = 1;
    byte y = 2 b;

    return;
}

int cwrvsgdhs(bool x1, bool x2, byte x0) {
    return 1;
    int x3;
    int x4;
    int x5;
    int x6;
    int x7;
    int x8;
    int x9;
    int xa;
    int xb;
    int xc;
    int xd;
    int xe;
    int xf;
    int x86 = 64;
}

int ytyjbd(bool x1, bool x2) {
    return 1;
}

int afsdtv(bool x1, bool x2) {
    return 1;
}

