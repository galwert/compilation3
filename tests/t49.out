---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
x1 BOOL -1
x2 BOOL -2
x INT 0
y BYTE 1
---end scope---
x1 BOOL -1
x2 BOOL -2
x0 BYTE -3
x3 INT 0
x4 INT 1
x5 INT 2
x6 INT 3
x7 INT 4
x8 INT 5
x9 INT 6
xa INT 7
xb INT 8
xc INT 9
xd INT 10
xe INT 11
xf INT 12
x86 INT 13
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
func ()->INT 0
ags ()->INT 0
zxvc ()->INT 0
sdg ()->INT 0
qwr (BOOL,BOOL)->INT 0
main ()->VOID 0
sdfa4 (BOOL,BOOL)->BOOL 0
nbfdrct (BOOL,BOOL)->VOID 0
cwrvsgdhs (BOOL,BOOL,BYTE)->INT 0
ytyjbd (BOOL,BOOL)->INT 0
afsdtv (BOOL,BOOL)->INT 0
